packages/client/src/api.ts

```diff
@@ -115,16 +198,35 @@ export abstract class MessagingChannelApi extends MessagingChannelBase {
     }, 0)
   }
 
+  /**
+   * When using converse, ends the answering turn of a message, terminating
+   * the waiting period and returning all payloads that were collected
+   * @param clientId id of the client that owns the message
+   * @param id id of the incoming message that has finished being answered
+   */
   async endTurn(clientId: uuid, id: uuid) {
```

Comment:
Isn't this function also mainly (or even solely) for internal use?

Reply:
Yes and no. It could theoretically be used by someone that wants to use converse with a custom channel (for whatever reason)